app/_posts/2024-11-05-kuma-2-9-0.md

```diff
@@ -0,0 +1,108 @@
+
+## New TLS policy
+
+Mutual TLS is one of the most common use cases in Kuma. This is because it's the base on which you can build fully secured  trust environments.
+The most common way to roll out mutual TLS is by first running services in **permissive mode,** in this mode, while the mesh will verify client certificates,
+it will still allow unsecured connection.
+You can then progressively remove unsecure connections by moving clients inside the mesh and once it's complete switch to **strict mode.**
+
+This has been long supported in Kuma. However, it was done at the mesh wide level which created an _all or nothing setup._
+In 2.9.x we've introduced a policy to apply this to only some data planes, thus enabling easier progressive adoption of strong security.
+
+To check how to do this please checkout the [MeshTLS guide](/docs/2.9.x/guides/progressively-rolling-in-strict-mtls/).
+
+This feature is part of a larger effort to address challenges users may be facing rolling out a service mesh in their production environment.
+We are always looking for more input on these challenges, if you are willing to share please feel free to reach out to us on [slack](https://kuma.io/community/).
+
+## New service resources 
+
+Kuma from the beginning have implicitly extracted services from its data planes.
+While this was convenient to get started it was coming with the following shortcomings:
+
+- It wasn't possible to have multiple services fronting the same port of a data plane, this is required for blue-green deployment for example.
+- Sub-setting services was impossible in policies which was making things hard to understand, observe and compose across multiple policies.
+- The opacity between multi-zone and single zone services was complex. We believe that while multi-zone services are useful the real necessity is cross zone communication. Making all services multi-zone by default was making a lot of simple cross-zone setups harder than they should have been.
+- You may want to manage traffic going across zones differently from traffic that is staying inside a zone.
+
+With `MeshService` we now have a dedicated service object which selects a set of data planes to send traffic to.
+This `MeshService` is defined in both policies and Envoy configuration, making it very easy to understand and observe.
+The resources can be labeled which enables policies that applies to whole subset of your services.
+
+[`MeshService`](/docs/2.9.x/networking/meshservice/) is cross-zone by default but not multi-zone.
+This means that each `MeshService` can only be present in a single zone at time.
+However, it can be accessed by other zones thanks to our cross-zone communication setup.
+If you want a service to be multi-zone you need to create a [`MeshMultizoneService`](/docs/2.9.x/networking/meshmultizoneservice/) which will group multiple services inside the same _pseudo-service._
```

Comment:
```suggestion
If you want a service to be multi-zone you need to create a [MeshMultizoneService](/docs/2.9.x/networking/meshmultizoneservice/) which will group multiple services inside the same _pseudo-service._
```
Using special formatting with links hides the information that this is link, I wasn't aware it was link until I moved my mouse over it by accident